docker-compose-dev.yml

```diff
@@ -2,14 +2,31 @@ version: "3.9"
 
 services:
 
-  codeql:
-    image: tpf_codeql
+  codeql_core:
+    image: tpf_codeql_core
+    build:
+      context: './SAST/codeql/core'
+      dockerfile: "./Dockerfile"
+    volumes:
+      - codeql_core_interface:/tp-framework/SAST/codeql/core
+
+  codeql_v2_9_2:
```

Comment:
I would comment this service out. We can use internally in our context to test the improvements of codeql versions